tracer/src/Datadog.Trace/Configuration/ImmutableExporterSettings.cs

```diff
@@ -70,5 +72,33 @@ public ImmutableExporterSettings(ExporterSettings settings)
         /// </summary>
         /// <seealso cref="ConfigurationKeys.MetricsPipeName"/>
         public string MetricsPipeName { get; }
+
+        /// <summary>
+        /// Gets the port where the DogStatsd server is listening for connections.
+        /// Default is <c>8125</c>.
+        /// </summary>
+        /// <seealso cref="ConfigurationKeys.DogStatsdPort"/>
+        public int DogStatsdPort { get; }
+
+        /// <summary>
+        /// Gets a value indicating whether partial flush is enabled
+        /// </summary>
+        public bool PartialFlushEnabled { get; }
+
+        /// <summary>
+        /// Gets the minimum number of closed spans in a trace before it's partially flushed
+        /// </summary>
+        public int PartialFlushMinSpans { get; }
+
+        /// <summary>
+        /// Gets the transport used to send traces to the Agent.
+        /// </summary>
+        internal TracesTransportType TracesTransport { get; }
```

Comment:
`TracesTransport` (with plural `Traces`) sounds weird to me. Shouldn't that be `TraceTransportType` and `TraceTransport`?

Reply:
I'm just trying to be consistent here, everything else is plural. 😆